src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from 'react';
+import { string, number } from 'prop-types';
+import { C_SHADOW } from '@bbc/psammead-styles/colours';
+import styled from 'styled-components';
+import { Headline } from '@bbc/psammead-headings';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import {
+  GEL_SPACING_QUAD,
+  GEL_SPACING,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
+import { formatUnixTimestamp } from '@bbc/psammead-timestamp-container/utilities';
+import { getBodyCopy } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const StyledHeadline = styled(Headline)`
+  @media (min-width: 1px) {
+    padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_TRPL};
+  }
+`;
+
+const BrandTitle = styled.span`
+  display: block;
+  padding-bottom: ${GEL_SPACING};
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    padding-bottom: 0;
+  }
+`;
+
+const Datestamp = styled.span`
+  ${({ script }) => script && getBodyCopy(script)};
+  ${({ service }) => getSansRegular(service)};
+  color: ${C_SHADOW};
+  margin: 0;
+`;
+
+const HeadingContainer = ({ idAttr, brandTitle, releaseDateTimeStamp }) => {
+  const { script, service, timezone, locale } = useContext(ServiceContext);
+
+  const formattedTimestamp = formatUnixTimestamp({
+    timestamp: releaseDateTimeStamp,
+    format: 'LL',
+    timezone,
+    locale,
+    isRelative: false,
+  });
+
+  return (
+    <StyledHeadline script={script} service={service} id={idAttr} tabIndex="-1">
+      <span
+        // eslint-disable-next-line jsx-a11y/aria-role
+        role="text"
```

Comment:
can this be an attribute on the h1 to save on nesting?

Reply:
Afraid not that would remove the h1 semantics

Reply:
Cheers - just gonna resolve this thread